addons/account/models/account_payment.py

```diff
@@ -869,240 +848,86 @@ def create(self, vals_list):
             # Hack to force a custom balance.
             force_balance_vals_list.append(vals.pop('force_balance', None))
 
-            # Force the move_type to avoid inconsistency with residual 'default_move_type' inside the context.
-            vals['move_type'] = 'entry'
+            # Hack to add a custom line.
```

Comment:
what kind of custom line? there's already 2 other 'custom' lines use cases before... :upside_down_face:

Reply:
The PoS for instance creates an entirely custom payment as well... Not just writeoff lines.
Just trying to not break anything, we will probably need to re-add some sync mechanism in the near future anyway

Reply:
mmkay... I don't have energy to fight on that anyway :sweat_smile: